api/v1alpha2/linodeobjectstoragebucket_webhook_test.go

```diff
@@ -58,25 +59,67 @@ func TestValidateLinodeObjectStorageBucket(t *testing.T) {
 					mck.LinodeClient.EXPECT().GetRegion(gomock.Any(), gomock.Any()).Return(&region, nil).AnyTimes()
 				}),
 				Result("success", func(ctx context.Context, mck Mock) {
+					bucket := bucket
+					bucket.Spec.Region = "iad"
 					assert.NoError(t, bucket.validateLinodeObjectStorageBucket(ctx, mck.LinodeClient))
 				}),
 			),
-		),
-		OneOf(
 			Path(
-				Call("invalid cluster format", func(ctx context.Context, mck Mock) {
+				Call("valid", func(ctx context.Context, mck Mock) {
 					region := region
 					region.Capabilities = slices.Clone(capabilities)
 					mck.LinodeClient.EXPECT().GetRegion(gomock.Any(), gomock.Any()).Return(&region, nil).AnyTimes()
 				}),
+				Result("success", func(ctx context.Context, mck Mock) {
+					bucket := bucket
+					bucket.Spec.Region = "us-iad"
```

Comment:
so both `us-iad` and `iad` are valid regions for buckets?

Reply:
and `us-iad-1` below?

Reply:
for now us-iad is the actual region with linode storage capabilities, but this is just to show that a single "lowercaseword" without hyphen can also pass through the **format validation** for future support.

`mck.LinodeClient.EXPECT().GetRegion(gomock.Any(), gomock.Any()).Return(&region, nil).AnyTimes()`
just mocks for now that 'iad' returns a valid region with linodeStorage capabilities.